lib/ecies/ecies.js

```diff
@@ -3,31 +3,56 @@
 var bitcore = require('../../')
 
 var PublicKey = bitcore.PublicKey
+var PrivateKey = bitcore.PrivateKey
 var Hash = bitcore.crypto.Hash
 var $ = bitcore.util.preconditions
-
-var AESCBC = require('./aescbc')
-
-// http://en.wikipedia.org/wiki/Integrated_Encryption_Scheme
-var ECIES = function ECIES (opts) {
+var bitcoreECIES = require('./bitcore-ecies')
```

Comment:
"bitcore" ecies seems like a poor name for this. fist of all it has "bitcore" in the name and this isn't bitcore. secondly, maybe we should just remove the other one if people are willing to standardize around this one, in which case it should just be called 'ecies'. or even just "ecies1" and "ecies2". or "ecies" and "ecies-electrum"

Reply:
I've asked myself the same question,but find no answer. Any advice?
The electrum-ecies can be called 'BIE1' as short of Bitcoin_Integrated_Encryption.
How about 'BIE0' for old bitcore-ecies?

Well, we should consider a BIP or something later to standardize it.

Reply:
after some thought i think the names should be:

- bitcore-ecies
- electrum-ecies
- index (points to electrum-ecies)